### PR TITLE
Fix: [Max2d.mod] Only require graphics instance if needed

### PR DESCRIPTION
**Pull request by GWRon**
In https://www.syntaxbomb.com/blitzmax-blitzmax-ng/revisiting-blitzmax-ng-trying-to-run-provided-samples/new/#new
"SToS" blamed that NG is no longer capable of running "samples/flameduck/oldskool2".

This was because in one of my recent max2d-changes I made some stuff a "method" and thus also relying on the procedural functions requiring an instance of TMax2DGraphics.

But for LoadImage() and its collegues, this is not the case - this PR changes things back to use "Functions" where only "globals" of the TMax2DGraphics-Type are used.
And this makes oldskool2 compile and run OK again.

TLDR:
This allows to use `LoadImage()` before `Graphics()` again (of course TImage still contains no valid imageframe then - but this is autohandled via the `seqs` property as it was done in legacy already).

### CODE REVIEW

**Comment by woollybah** [comment]
And globals are the best thing to use?

Or would be better just to make the sample work with the changes?

**Comment by GWRon** [comment]
The "globals" are still in there - as they were in original max2d.

The only change is that I do not call a "method" but a "function" (content is the same) and this avoids requiring a "getCurrent()" (which would be "null").
